.gitignore

```diff
@@ -18,6 +18,7 @@ lib/
 lib64/
 parts/
 sdist/
+weights/
```

Comment:
Can you put this at the bottom of the file? I made a section at the bottom for non-standard .gitignore entries.

Reply:
Done